corehq/apps/reports/scheduled.py

```diff
@@ -1,11 +1,15 @@
 from calendar import monthrange
 from datetime import datetime
 from corehq.apps.reports.models import ReportNotification
+from corehq.util.soft_assert import soft_assert
+
+
+_soft_assert = soft_assert(exponential_backoff=True)
 
 
 def get_scheduled_reports(period, as_of=None):
     as_of = as_of or datetime.utcnow()
-    assert period in ('daily', 'weekly', 'monthly')
+    assert period in ('daily', 'weekly', 'monthly'), period
```

Comment:
Ha. cory linked the commit for some context in the interrupt ticket. figured I'd just follow up. Don't think it's ever been triggered